backend/routes/db.py

```diff
@@ -365,21 +365,9 @@ def get_researchers(ids: List[str] = Query(...), fields: Union[List[str], None]
 
 @router.get("/db-refresh")
 def db_refresh_route():
-    """Triggers refresh of the database
-    todo: May want to change this back to GH action for reasons:
```

Comment:
## Rationale
Wrote a bunch of comments here for these reasons.
I think another one is 'server memory', but I don't think that was listed there.